### PR TITLE
Old UI: Delete It!

### LINKED ISSUE #267: Old UI: delete it
The new React UI is close enough to feature parity that forcing everyone to switch is very low cost.

Do This
=====
* Delete the old UI
  * `public/`
     * Move day/night theme files over to `frontend/`
  * `spec/` (I think)
* Make new frontend available at `/` instead of `/frontend` 
* Close all issues that no longer apply

### PR DESCRIPTION
**Pull request by danielbeardsley**
Good night sweet prince. React + Chakra has replaced you.

Closes #267

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,8 +66,8 @@
     "frontend:start": "DUMMY_USER=danielbeardsley DUMMY_PULLS=./frontend/dummy-pulls.json webpack-dev-server --config ./frontend/webpack.dev.config.js",
     "frontend:watch": "webpack --watch --config ./frontend/webpack.dev.config.js",
     "frontend:build": "webpack --config ./frontend/webpack.prod.config.js",
-    "postinstall": "webpack && npm run frontend:build",
-    "build": "webpack && npm run frontend:build",
+    "postinstall": "npm run frontend:build",
+    "build": "npm run frontend:build",
```
Comment (jarstelfox) [inline package.json:70]: Should we delete `frontend:build` in favor of `build`?

**Review by jarstelfox — APPROVED** [review]
CR 🌵

File: package.json (hunk 1 above)
Comment (danielbeardsley) [reply package.json:70]: I'm fine with transitioning these task names for sure.

**Comment by deltuh-vee** [comment]
QA 🎬 
Old UI is gone now.

File: package.json [hunk 2]
```diff
@@ -31,7 +28,6 @@
     "passport-github2": "^0.1.12",
     "popper.js": "^1.16.1",
```
Comment (ardelato) [inline package.json:28]: I ran `npx depcheck` and found that `popper.js` is also unused now

**Comment by ardelato** [comment]
CR 👍 with a question about possibly another unused npm package.

File: package.json (hunk 2 above)
Comment (ardelato) [reply package.json:28]: A heads up though, that tool also mentioned other `devDependencies` and `dependencies` that were unused but I found some uses for the other ones mentioned.

```
Unused dependencies
* @types/underscore
* popper.js
Unused devDependencies
* eslint-plugin-import
* husky
* webpack-cli
```

Comment (danielbeardsley) [reply package.json:28]: Fixed.

**Review by ardelato — APPROVED** [review]
CR 👍

**Review by jarstelfox — APPROVED** [review]
CR 🌵

File: package.json (hunk 2 above)
Comment (danielbeardsley) [reply package.json]: Thanks! One of those was unused, the others were used, but just not directly by name.

**Comment by jarstelfox** [comment]
CR 🌵

**Review by ardelato — APPROVED** [review]
CR 👍

**Pull request merged**